<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>561</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>3D-BattleShip</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="reset_attack_from_player2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>470</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="player" name="player_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="player" name="player_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>401</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_attack_from_player1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>470</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attack_from_player2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>470</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attack_from_player1">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>attack</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_player1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt; Player 1 &gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_player2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt; Player 2 &gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>421</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="message_for_player1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>500</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="message_for_player2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>470</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_player1_attack_command">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>470</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[ Player 1 Attack Command ]</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_player2_attack_command">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[ Player 2 Attack Command ]</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_player2_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_player1_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_from_player2_to_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(from player 2 to 1)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_from_player1_to_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>500</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>(from player 1 to 2)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="master" name="gamemaster">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>480</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>graphicsView_4</zorder>
   <zorder>graphicsView_3</zorder>
   <zorder>graphicsView_2</zorder>
   <zorder>graphicsView</zorder>
   <zorder>reset_attack_from_player2</zorder>
   <zorder>player_1</zorder>
   <zorder>player_2</zorder>
   <zorder>reset_attack_from_player1</zorder>
   <zorder>attack_from_player2</zorder>
   <zorder>attack_from_player1</zorder>
   <zorder>label_player1</zorder>
   <zorder>label_player2</zorder>
   <zorder>message_for_player1</zorder>
   <zorder>message_for_player2</zorder>
   <zorder>label_player1_attack_command</zorder>
   <zorder>label_player2_attack_command</zorder>
   <zorder>label_player2_2</zorder>
   <zorder>label_player1_2</zorder>
   <zorder>label_from_player2_to_1</zorder>
   <zorder>label_from_player1_to_2</zorder>
   <zorder>gamemaster</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>player</class>
   <extends>QOpenGLWidget</extends>
   <header>player.h</header>
  </customwidget>
  <customwidget>
   <class>master</class>
   <extends>QLabel</extends>
   <header>master.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
